pyproject.toml

```diff
@@ -33,3 +33,40 @@ exclude = '''
   | treeherder/changelog/migrations
 )/
 '''
+
+[tool.ruff]
+# Same as Black.
+line-length = 100
+
+# Assume Python 3.9
+target-version = "py39"
+
+# In addition to the standard set of exclusions, omit all tests, plus a specific file.
+extend-exclude = ["*/.*/",".*/","__pycache__","node_modules"]
+
+select = [
+  # pycodestyle
+  "E",
+  "W",
+  # pyflakes
+  "F",
+]
+
+ignore = [
+  # E203 whitespace before ':'
+  "E203",
+  # E231: missing whitespace after ','
+  "E231",
```

Comment:
These rules are not triggered right now so I'm not sure it's still relevant to ignore them.

Reply:
Done in https://github.com/mozilla/treeherder/pull/7898/commits/5146fb4adbb4ba79d9f571d2eec6fea196978493